proxy/src/registry.rs

```diff
@@ -348,6 +348,37 @@ impl Registry {
         }))
     }
 
+    /// List orgs of the Registry.
+    ///
+    /// # Errors
+    ///
+    /// Will return `Err` if a protocol error occurs.
+    pub async fn list_orgs(&self, _user_id: String) -> Result<Vec<Org>, error::Error> {
+        // TODO(merle): Remove temp_public_key once members are returned as user ids
+        let temp_public_key = ed25519::Pair::from_legacy_string("//Alice", None).public();
+        let org_ids = self.client.list_orgs().await?.into_iter();
+        let mut orgs = Vec::new();
+        for org_id in org_ids {
+            orgs.push(self.client.get_org(org_id).await?.expect("Get org"));
```

Comment:
Strange.. does the client return a nested `Result`?

Reply:
@cloudhead Yes it does, they are for different stages of the transaction lifecycle.